### PR TITLE
Alignment / spacing fix - Closes #875

### LINKED ISSUE #875: Network monitor alignment / spacing
As mentioned in #868 the spacing between elements can be improved for particular views, esp. network monitor.

grafik

_Originally posted by @punkr0ck in https://github.com/LiskHQ/lisk-explorer/issues/868#issuecomment-449365844_

### PR DESCRIPTION
**Pull request by youxiberlin**
### What was the problem?
- Spacing between elements needed to be improved for the particular views.
- Spacing of the tabs needed to be aligned to the other elements for some pages.
- The bottom line of the currency picker in the breadcrumbs was visible. 

### How did I fix it?
- Added margin at the upper section of Delegate Monitor.
- Set margin-left of the first tab of the nav-tabs.
- Set height and margin-bottom at elements in the nav-tabs to hide the bottom lines.
- Changed the margin-bottom of the currency picker in the breadcrumb to hide the bottom line.

### How to test it?
- Checked on browser.

### Review checklist

* The PR solves #875 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the
	[commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: src/assets/styles/common.css [hunk 1]
```diff
@@ -1760,3 +1760,25 @@ li.dropdown.disabled a {
...
+  }
+
+  .nav-tabs > li:first-child > a {
+    margin-left: 32px;
+  }
+}
+
+.nav-tabs > li > a {
+  height: 45px;
+  margin-bottom: -1px;
```
Comment (MichalTuleja) [inline src/assets/styles/common.css]: Please try with the 0px again, it does not look so well.

File: src/assets/styles/common.css [hunk 2]
```diff
@@ -1760,3 +1760,25 @@ li.dropdown.disabled a {
...
+}
+
+.exchanges .nav-tabs > li.active > a  {
+  height: auto;
+}
+
+.market-watcher .nav-tabs > li:first-child > a {
+  margin-left: 0px;
+}
```
Comment (MichalTuleja) [inline src/assets/styles/common.css]: Newline

File: src/assets/styles/common.css [hunk 3]
```diff
@@ -1760,3 +1760,62 @@ li.dropdown.disabled a {
...
+
+@media (min-width: 769px) and (max-width: 1024px) {
+  .latest-votes-table {
```
Comment (MichalTuleja) [inline src/assets/styles/common.css]: The latest votes table does not align well when displayed in a 425px width.

Comment (youxiberlin) [reply src/assets/styles/common.css]: Modified html and css to make the table for Latest Votes and Newest Delegates consistent.

**Pull request merged**